virttest/utils_v2v.py

```diff
@@ -25,7 +25,11 @@
 
 from virttest import data_dir
 from virttest import libvirt_vm as lvirt
-from virttest import ovirt, ppm_utils
```

Comment:
How about ppm_utils module? Is it not necessary as well?

Reply:
v2v don't use it any more.